composer.json

```diff
@@ -70,10 +70,12 @@
         ],
         "post-install-cmd": [
             "Illuminate\\Foundation\\ComposerScripts::postInstall",
-            "php artisan storage:link"
+            "php artisan storage:link",
+            "php artisan vendor:publish --provider=\"Backpack\\FileManager\\FileManagerServiceProvider\" --tag=views --force"
         ],
         "post-update-cmd": [
-            "Illuminate\\Foundation\\ComposerScripts::postUpdate"
+            "Illuminate\\Foundation\\ComposerScripts::postUpdate",
+            "php artisan vendor:publish --provider=\"Backpack\\FileManager\\FileManagerServiceProvider\" --tag=views --force"
```

Comment:
```suggestion
```

I think it's enough to have it in the `install` cmd. You shouldn't be running `update` in production environment.